biomes/abstract/terrain/land/mountains-large/eq_serrated_coast.yml

```diff
@@ -0,0 +1,28 @@
+id: EQ_SERRATED_COAST
+type: BIOME
+abstract: true
+extends: NO_TERRAIN_2D
+
+# Large smooth stylized mountains with lots of overhangs.
```

Comment:
Lol forgot to remove this line from the config file, should probably do that sometime in the future, or change it to something else.